interfaces/Portalicious/src/app/domains/project/project.api.service.ts

```diff
@@ -130,7 +130,7 @@ export class ProjectApiService extends DomainApiService {
       params: {
         includeProgramRegistrationAttributes,
         includeTemplateDefaultAttributes,
-        filterShowInPeopleAffectedTable,
+        filterShowInRegistrationsTable,
```

Comment:
This is changing what is sent to the backend, but I don't see any change in the backend to account for the new attribute name